docs/content/doc/usage/issue-pull-request-templates.en-us.md

```diff
@@ -39,3 +39,6 @@ Possible file names for PR templates:
 * .gitea/pull_request_template.md
 * .github/PULL_REQUEST_TEMPLATE.md
 * .github/pull_request_template.md
+
+
+Additionally, the New Issue page URL can be suffixed with `?body=Issue+Text` and the form will be populated with that string. This string will be used instead of the template if there is one.
```

Comment:
Doesn't this also work for pull requests? If so, you should mention that :slightly_smiling_face:

Reply:
It does not, this PR adds supports only for new issues. Implementing this to new pull requests makes little sense to me, as the developer also has to know which branches the user needs to compare in advance.

Reply:
Makes sense, thanks for clarifying!